lib/reverse_string.rb

```diff
@@ -1,5 +1,16 @@
 def reverse_characters(str)
+    #create an array dividing up the string into individual characters
+    array = [str.split("")]
```

Comment:
Since `str.split("")` itself returns an array, you are **doubling up** on arrays here. Because you're taking the array that `str.split("")` gives you _and then putting that_ inside of the `[` and `]` characters. So you'll end up with this:

```ruby
[["H", "e", "l", "l", "o"]]
```

(Notice the extra square brackets on the ends.

---

This line should simply be `array = str.split("")`–no extra square brackets.

I'm going to give more feedback below under the presumption of this fix being in place.